Content.Client/Medical/CrewMonitoring/CrewMonitoringWindow.xaml.cs

```diff
@@ -90,14 +92,15 @@ private BoxContainer GetPositionBox(MapCoordinates? coordinates, Vector2 sensorP
             {
                 // todo: add locations names (kitchen, bridge, etc)
                 var pos = (Vector2i) coordinates.Value.Position;
+                var mapCoords = coordinates.Value.ToMap(_entityManager).Position;
                 var dirIcon = new DirectionIcon(snap, precision)
                 {
                     SetSize = (IconSize, IconSize),
                     Margin = new(0, 0, 4, 0)
                 };
                 box.AddChild(dirIcon);
                 box.AddChild(new Label() { Text = pos.ToString() });
-                _directionIcons.Add((dirIcon, coordinates.Value.Position - sensorPosition));
+                _directionIcons.Add((dirIcon, mapCoords - sensorPosition));
```

Comment:
Is this correct? Just doing mapcoords - mapcoords won't give the correct rotation in your frame of reference.

Reply:
Seems like it's correct. The other way wasn't giving me the correct position.